src/test/java/com/artipie/asto/KeyTest.java

```diff
@@ -78,4 +78,20 @@ void keyWithEmptyPart() {
     void resolvesRootKey() {
         MatcherAssert.assertThat(Key.ROOT.string(), Matchers.equalTo(""));
     }
+
+    @Test
+    void returnsParent() {
```

Comment:
@brastak let's test scenario which will return empty parent for `new Key.From("")`